src/ui/components/ErrorContent/ErrorContent.js

```diff
@@ -272,12 +276,19 @@ class ErrorContent extends React.PureComponent {
         }
 
         return (
-            <div className={b({'no-controls': noControls, 'is-mobile': DL.IS_MOBILE}, className)}>
+            <div
+                className={b(
+                    {'no-controls': noControls, 'is-mobile': DL.IS_MOBILE, size},
+                    className,
+                )}
+            >
                 <PlaceholderIllustration
                     name={imageName}
                     title={this.renderTitle()}
                     description={this.renderDescription()}
                     renderAction={() => this.renderActions()}
+                    size={size}
+                    direction={direction}
```

Comment:
By default in `PlaceholderIllustration` direction switches to column in mobile view. This behavior may get lost if we pass direction in that way. I think you need to check here by DL.IS_MOBILE too

Reply:
Thank you for the remark. Will add this behavior.